sig-and-wg/sig-core/proposals/authorization-proxy.md

```diff
@@ -0,0 +1,39 @@
+# Installation on managed cluster
+
+Created on 2018-08-02 by Piotr Bochyński (@pbochynski).
+
+## Status
+
+Proposed on 2018-08-02
+
+## Abstract
+
+It is not possible to install the Kyma project on the managed clusters such as Google Kubernetes Engine (GKE) or Azure Kubernetes Service (AKS). To enable the Kyma project adoption, it must run on any infrastructure supporting Kubernetes. Kyma requires access to the kube-apiserver configuration and uses some Kubernetes alpha features, which is **not allowed** in many managed clusters. Even if some providers allow such configuration, it must happen during the cluster provisioning, which makes Kyma installation process more complex.
+
+## Problems to solve
+
+1. The kube-apiserver component must be configured to verify tokens coming from an external OIDC provider (coming with Kyma). Usually, it is done during the installation by providing additional parameters such as **oidc-issuer-url**, **oidc-client-id**, **oidc-username-claim**, to name a few. These parameters **cannot** be set in GKE or AKS.
+2. Kyma depends on some alpha features of Kubernetes that are not supported by the managed clusters, for example PodPreset from `settings.k8s.io/v1alpha1`. Other apiserver configuration entries required by kyma, that might be missing in managed clusters:
+    - admission plugins: Initializers, NamespaceLifecycle, LimitRanger, ServiceAccount, MutatingAdmissionWebhook, ValidatingAdmissionWebhook, DefaultStorageClass, ResourceQuota, PodPreset
+    - alfa APIs : batch/v2alpha1, settings.k8s.io/v1alpha1, admissionregistration.k8s.io/v1alpha1
+    
+## Proposal
+
+### OIDC configuration
+
+> Read [this](https://github.com/kyma-project/kyma/blob/master/docs/authorization-and-authentication/docs/003-architecture.md) document first to understand the current authorization concept.
```

Comment:
`>**NOTE:** Read [this](https://github.com/kyma-project/kyma/blob/master/docs/authorization-and-authentication/docs/003-architecture.md) document first to understand the current authorization concept.`

Reply:
done